examples/pylint/C0121_singleton_comparison.py

```diff
@@ -0,0 +1,14 @@
+"""pylint: singleton comparison
+
+"""
+
+def is_true():
+    """
+    @type obj: int
+    @rtype: bool
+    """
+    temp = 5
+    if temp == True:
+        return False
```

Comment:
Please check whether the condition `if temp is None` gives this same warning. If it does, disable this warning.